syntax = "proto3";

package keto.proto;

/**
 * This is an example of a protobuf message.
 */
message ContractMessage {
    bytes account_hash = 1;
    bytes contract_hash = 2;
    string contract_name = 3;
    bytes contract_owner = 4;
    string contract = 5;
    string contract_namespace = 6;
}
